<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSettings</class>
 <widget class="QDialog" name="DlgSettings">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>385</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>385</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>290</width>
    <height>385</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DlgSettings {
  background-color: rgb(28,52,62);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBoxAppServer">
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="title">
        <string>Application Server</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color: white;</string>
          </property>
          <property name="text">
           <string>Server Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditServer">
          <property name="styleSheet">
           <string notr="true">color: black;</string>
          </property>
          <property name="maxLength">
           <number>255</number>
          </property>
          <property name="placeholderText">
           <string>&lt;Server Address&gt;:&lt;Port&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxLogin">
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="title">
        <string>Credentials</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>User login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditLogin">
          <property name="styleSheet">
           <string notr="true">color: black;</string>
          </property>
          <property name="maxLength">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPassword">
          <property name="styleSheet">
           <string notr="true">color: black;</string>
          </property>
          <property name="maxLength">
           <number>255</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Let empty to take stored password.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxRememberPw">
          <property name="toolTip">
           <string>Remember your login and sign in automatically.</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Sign in on application start</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButtonSignIn">
            <property name="toolTip">
             <string>Sign in into your account.</string>
            </property>
            <property name="styleSheet">
             <string notr="true">color: black;</string>
            </property>
            <property name="text">
             <string>Sign in</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSignOut">
            <property name="toolTip">
             <string>Sign out from your account.</string>
            </property>
            <property name="styleSheet">
             <string notr="true">color: black;</string>
            </property>
            <property name="text">
             <string>Sign out</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSignIn</sender>
   <signal>clicked()</signal>
   <receiver>DlgSettings</receiver>
   <slot>onBtnSignInClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSignOut</sender>
   <signal>clicked()</signal>
   <receiver>DlgSettings</receiver>
   <slot>onBtnSignOutClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtnSignInClicked()</slot>
  <slot>onBtnSignOutClicked()</slot>
 </slots>
</ui>
